src/NuGet.Core/NuGet.Packaging/PackageReaderBase.cs

```diff
@@ -542,5 +545,51 @@ public virtual Task<string> CopyNupkgAsync(string nupkgFilePath, CancellationTok
         {
             throw new NotImplementedException();
         }
+
+        public async Task<IReadOnlyList<Signature>> GetSignaturesAsync(CancellationToken token)
+        {
+            var signatures = new List<Signature>();
+
+            var files = await GetFilesAsync(token);
+            var sigFile = files.FirstOrDefault(e => StringComparer.Ordinal.Equals(e, "testsigned/signed.json"));
```

Comment:
Perhaps turn `"testsigned/signed.json"` into a constant and reuse here and in [`SignPackageArchive.TestSignedPath`](https://github.com/NuGet/NuGet.Client/pull/1728/files#diff-601d481c80306a045d31673326332777R16)? 

I know this is WIP, but want to make sure :) Where does `testsigned/signed.json` come from? (maybe I missed a spec? or is this just a mock-up?)

Reply:
This is very fake signing for now, it will be thrown away. It just gives a way to use the APIs and write tests until the actual signing is added.

Reply:
Gotcha! 👍